angel_system/impls/detect_activities/two_stage/two_stage.py

```diff
@@ -28,20 +24,10 @@ def __init__(self, checkpoint: str, num_classes: int):
         # Temporal: (Dict) Input feature from ResNext (2048) + Aux_data ->
         # Output (1, num_classes)
         self.temporal = RULSTM(num_classes, hidden=128, dropout=0, depth=3)
-        check = torch.load(checkpoint)
-        
         self.fcn.eval()
         
         # Load checkpoint
-        temp_check = dict()
-
-        for key in check['state_dict'].keys():
-            #extract just the temporal weights from the checkpoint
-            #fcn doesn't change, so no need to load in weights for it
-            if 'temporal' in key:
-                new_key = key.split('temporal.')[1]
-                temp_check[new_key] = check['state_dict'][key]
-        self.temporal.load_state_dict(temp_check)
+        self.temporal.load_state_dict(torch.load(checkpoint))
```

Comment:
We should comment here that we expect to receive weights for the temporal layers only.